### ISSUE TITLE
Equation labels are left out when rendered under SVG

### ISSUE THREAD

**Issue by legendre6891** [mathjax/MathJax#1204]
In Chrome and Safari, when printing (e.g. Ctrl-P) the output, equation labels are not printed if the equations are rendered via SVG. 

This works correctly in Firefox (and equation labels are printed in other rendering modes).

**Comment by pkra** [comment]
Thanks for the report. Could you provide a live sample where a problem occurs?

I suspect this is unfortunately expected behavior (but again, a live sample would clarify). Say I open [this sample](http://cdn.mathjax.org/mathjax/latest/test/sample-eqnum.html) on my machine. Everything looks good. However, my viewport size is considerably wider than a print page. To print this page, Chrome has style the page to fit a print page (I don't know if they do more than apply a print stylesheet for this). 

While the HTML/CSS output can use relative widths, SVG does not have that notion, so MathJax has to hardcode the width between the label and the equation (the width is calculated during rendering to fit the viewport width). As a result, if you modify the viewport size after MathJax SVG rendering is done, you'll cut off the lable -- and this is what happens during print-to-pdf.

What would have to happen is for MathJax to re-render the content once Chrome applies the styling it uses for printing. That way it could recalculate the width between equation and label under the constraints of Chrome's (or the page's) print stylesheet. 

I don't know if there is a way to do so (besides generating a separate print version of the page with fixed width of a print page).

**Comment by legendre6891** [comment]
Ah I understand the problem -- that is too bad, but thanks for the explanation. (A live sample is available on my personal website here: http://www.kevinli.co/minimax-duality.html)

**Comment by pkra** [comment]
I think the following should work in theory -- but something goes wrong. Maybe ask on StackOverflow etc?

```javascript
var mediaQueryList = window.matchMedia('print');
mediaQueryList.addListener(function(mql) {
    if (mql.matches) {
      MathJax.Hub.Queue(["Rerender",MathJax.Hub]);
    } else {
        MathJax.Hub.Queue(["Rerender",MathJax.Hub]);
    }
});
``
